snap7/util.py

```diff
@@ -279,10 +336,11 @@ def parse_specification(db_specification):
     parsed_db_specification = OrderedDict()
 
     for line in db_specification.split('\n'):
-        if line and not line.startswith('#'):
-            row = line.split('#')[0]  # remove trailing comment
-            index, var_name, _type = row.split()
-            parsed_db_specification[var_name] = (index, _type)
+        if (not line.isspace()) and (len(line) != 0):
```

Comment:
Line 339 and 340 could be a one liner.
- With len(line)!=0 you say, that the string exist and is not empty, the same as `not line.
- With line.isspace() you check if there is a space, \n, \t and more.
- maybe a small boost (haven't meassured it), but you can replace `line.startswith('#')` with `line[0]=='#'` I would bet this saves some time and ressources.

So you could change that to `if line and not (line.isspace() or line[0]=='#')`
This would check, if line is >1, is not filled with spaces etc and does not start with an '#'